lib/commands/smapi/cli-customization-processor.js

```diff
@@ -40,7 +40,7 @@ class CliCustomizationProcessor {
     }
 
     _processBodyParameter(parameter, parentOperation, definitions) {
-        const rootName = camelCase(parameter.name);
```

Comment:
dont need to do camelCase. The logic was failing in one case when body parameter was not camel case and was starting with a capital letter.